markdown/chapters/entombsday/deck-construction.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
This chapter still feels kinda small in general. Maybe comments from the technical folks will reveal more on what could be added?

Reply:
Completely redid this chapter. Let me know what you think!